src/test/java/tests/Smoke.java

```diff
@@ -0,0 +1,91 @@
+package tests;
+
+import java.util.concurrent.TimeUnit;
+import common.AbstractComponent;
+import io.github.bonigarcia.wdm.WebDriverManager;
+import org.openqa.selenium.*;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.interactions.Actions;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+import pages.*;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+public class Smoke {
+
+    WebDriver driver;
+    WebDriverWait wait;
+    /**
+     * Constructor
+     */
+
+    /**
+     * Set up browser settings and open the Web Page
+     */
+    @BeforeMethod
+    public void setUp() throws InterruptedException{
+
+        WebDriverManager.chromedriver().setup();
+        driver = new ChromeDriver();
+        driver.manage().timeouts().implicitlyWait(10, TimeUnit.SECONDS) ;
+        driver.manage().window().maximize();
+    }
+
+    @AfterMethod
+    public void tearDown()  {
+        driver.quit();
+    }
+
+    @Test
+    public void shouldSuccessfullyPurchaseTheItem() throws InterruptedException{
+
+        AbstractComponent abstractComponent = new AbstractComponent(driver);
+        HomePage homePage = new HomePage(driver);
+        ProductsPage productsPage = new ProductsPage(driver);
+        CartPage cartPage = new CartPage(driver);
+        InformationPage informationPage = new InformationPage(driver);
+        ShippingPage shippingPage = new ShippingPage(driver);
+        PaymentPage paymentPage = new PaymentPage(driver);
+
+        homePage.goTo();
+
+        homePage.getPopUpButtonClose1();
```

Comment:
create a method for handling popups on homepage like "closeIfAnyPopupDisplayed()"